core/trino-main/src/main/java/io/trino/client/direct/DirectTrinoClient.java

```diff
@@ -21,6 +21,7 @@
 import io.trino.dispatcher.DispatchQuery;
```

Comment:
> When a query being executed by DirectTrinoClient takes longer than the query.client.timeout,
the query is then purged.

It is not "purged". The engine marks this query as abandoned by client and cancels it.

Reply:
i will adjust comment to `canceled` instead of `purged`